stracer/src/main/scala/com/evolutiongaming/stracer/Tracer.scala

```diff
@@ -89,10 +96,19 @@ object Tracer {
     implicit val configReaderStartupConf: ConfigReader[StartupConf] = deriveReader
   }
 
+  def defaultProbability: Option[Sampling] => Double = {
+    case Some(Accept) | Some(Debug) => 1.0
+    case _ => 0.01
```

Comment:
please unlatch all cases and avoid using `_` as in case of adding new cases - this code should be revisited